app/views/submissions/show.erb

```diff
@@ -6,6 +6,9 @@
     </h1>
 
     <div class="pull-right submission-toolbar hidden-xs">
+      <% if submission.user_exercise.help_requested? %>
+        <span class="label label-danger">Requested help</span>
```

Comment:
@Insti Thanks for your feedback. It is a good thing to know when working on other CSS components in future :)